service.librarywatchdog/addon.xml

```diff
@@ -17,8 +17,13 @@
             <icon>icon.png</icon>
         </assets>
         <news>
+v1.1.0
```

Comment:
try not to use news as a changelog, the ideia is not to be a complete replacement :) Next version though